### ISSUE TITLE
Matrix Edit+ breaks the styles within Content Container Templates

### ISSUE THREAD

**Issue by davidjb** [jcu-eresearch/matrix-helpers#45]
The following styles cause bulleted lists in content container templates to lose their bullets when displayed in Edit+ (from `easyedit.min.css`).  This wants investigating because there could be (probably are) lots more conflicts.  I wonder why it was done like this...

```css
.content_template_custom_layout ul {
  list-style-type: none !important;
}
```

**Comment by davidjb** [comment]
Reported at https://squizmap.squiz.net/matrix/10466.  Comments:

> The following CSS is included with Edit+ for Matrix 5.3.3.0; the styles conflict with any Content Container Templates that have a content editing area configured within its Simple Edit Layout (eg like in the documentation).  The result is that any bulleted lists (`<ul>`) see their list markers become invisible and for any links (`<a>`), their colouring is forced to being the same colour as the text (eg all text and links look black).  The cause is the following part of easyedit.min.css:

```css
.content_template_custom_layout ul {
list-style-type: none!important;
}

.content_template_custom_layout a {
color: inherit!important;
}

.content_template_custom_layout li {
margin: 0;
padding: 0 0 5px 0;
line-height: inherit;
}
```
> Removing these lines don't appear to have any effect on the Content Container and fix the above-mentioned issues.

In short, either get rid of the above CSS or override it and you'll be sorted.

**Comment by davidjb** [comment]
Sorted in 5.4.  That's been installed already.
